### PR TITLE
Align npm package with main repo

### PR DESCRIPTION
**Pull request by surma**
Hey @1000ch 🎉 

Before I start, thanks for building this npm package! It’s super convenient to have. Once we align our efforts, I’ll add your npm package in the main repo’s README, if you don’t mind.

This is a pretty brutal PR, but I hope we can work together to make this happen :)

Things I have done:

* Remove compilation fallback (for now). Most Node developers don’t have Go installed anways, so it’s a fallback that will rarely work. I am pretty confident that the binaries I provide on the main repo are sufficient. Additionally, `simplehttp2server` currently doesn’t have a `--version` flag, so it would always try to compile as it detects this as a bad binary.
* Download binaries from the main repo. This way you don’t need to commit them to your repo and if the main repo gets a new release you can just bump your repo and everything “should just work” ™️ 
* Align versioning of your npm package with the main repo’s version. It’s currently confusing that simplehttp2server v1.0.0 on npm installs simplehttp2server v3.1.3. I am using the version string to automatically download binaries of the same version.

Let me know what you think :)

### CODE REVIEW

**Comment by 1000ch** [comment]
@surma Thank you for PR at first 👍 

> Align versioning of your npm package with the main repo’s version. It’s currently confusing that simplehttp2server v1.0.0 on npm installs simplehttp2server v3.1.3. I am using the version string to automatically download binaries of the same version.

It's simple and relevant but sometimes confusing. When should be patched for a problem such as Node.js deps issue but simplehttp2server itself, I cannot release a patch version.

**Comment by surma** [comment]
Fair point. I’ll remove the version bump for now :)

File: lib/index.js [hunk 1]
```diff
@@ -4,13 +4,21 @@ const path = require('path');
 const BinWrapper = require('bin-wrapper');
 const pkg = require('../package.json');
 
-const url = `https://raw.github.com/1000ch/simplehttp2server/v${pkg.version}/vendor/`;
+const url = `https://github.com/GoogleChrome/simplehttp2server/releases/download/${pkg.version}/`;
```
Comment (1000ch) [inline lib/index.js]: Should be `https://github.com/GoogleChrome/simplehttp2server/releases/download/3.1.3/`?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "simplehttp2server",
-  "version": "1.0.0",
+  "version": "2.0.0",
```
Comment (1000ch) [inline package.json]: I'll bump by myself when I publish, so please discard this change 🙆

**Comment by surma** [comment]
@1000ch Please take a look. Also, I totally missed that I duplicated the logic of `suffix.js`. I fixed that as well :)

**Comment by 1000ch** [comment]
🙌

**Comment by surma** [comment]
🎉 🎉 🎉 🎉

**Pull request merged**